### DISCUSSION TITLE
Wordpress Multisite: Remove/Hide OpenID plugin menu

### DISCUSSION THREAD

**Discussion by s0n1kete** [oidc-wp/openid-connect-generic#490] [Q&A] [answered]
Hello,

I would like to know if it is possible to hide the plugin in the wordpress menu and only make it visible to super administrators. I have been trying with remove_submenu_page and remove_menu_page but I don't know what would be the slug that I should put.
I want that only the super admins of the multisite to be able to edit it

Thanks you

**Comment by timnolte** [comment]
@s0n1kete I thought I had replied here but this will most likely need some development to add a new capability that can be used. You can find the slug easily enough in the code, it is a constant defined in the plugin code.

**Reply by s0n1kete** [reply]
Hello @timnolte , yes but I don't want to modify the plugin code for possible updates, I prefer to do it through another second plugin with hooks

**Reply by timnolte** [reply]
@s0n1kete I'm not saying to update your installed version of the plugin, I'm saying this will need to be changed on the plugin by either someone submitting a pull request or me working on the changes.

**Reply by timnolte** [reply, marked as answer]
@s0n1kete you'll find the name of the settings page here:

https://github.com/oidc-wp/openid-connect-generic/blob/dev/includes/openid-connect-generic-settings-page.php#L49

The call to add the page is here: https://github.com/oidc-wp/openid-connect-generic/blob/dev/includes/openid-connect-generic-settings-page.php#L107

So if you want to remove the page for a certain role you'll just need to make sure your call happens later than the call where the settings page is added.

**Reply by s0n1kete** [reply]
okey, thanks you
